cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/using-devfiles-in-odo.adoc

```diff
@@ -15,6 +15,11 @@ include::modules/developer-cli-odo-about-devfiles-in-odo.adoc[leveloffset=+1]
 * You have installed `{odo-title}`. 
 * You must know your ingress domain cluster name. Contact your cluster administrator if you do not know it. For example, `apps-crc.testing` is the cluster domain name for https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/[Red Hat CodeReady Containers].
 
+[NOTE]
+====
+Currently odo does not support creating devfile components with `--git` or `--binary` flags. You must use S2I component flow to create components when using these flags.
```

Comment:
Should we help readers to understand what a "component flow" is? Should "component flow" have "a" or "the" in front?

Reply:
I've reworded is as 
>Currently odo does not support creating devfile components with `--git` or `--binary` flags. You can only create S2I components when using these flags.

(Dropped the "flow" since it sounded somewhat jargony in this context.)